### ISSUE TITLE
Problem with "migrate" function from autosubmit

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#535]
In GitLab by @thomasarsouze on Mar 3, 2020, 12:49

Hi,

As stated in the title, I want to migrate my a28v experiment from RES account to PRACE account.

I followed the instructions [here](https://autosubmit.readthedocs.io/en/latest/usage/migrate.html). 

Here is the log from the *offer* followed by the *pickup*:
```Migrating experiment a28v

Checking configuration files...
platforms_a28v.conf OK
jobs_a28v.conf OK
autosubmit_a28v.conf OK
expdef_a28v.conf  OK
^[[32mConfiguration files OK
^[[0m^[[39m
Checking remote platforms
Checking [marenostrum4] from platforms configuration...
Project in platform configuration file successfully updated to cns61
Host in platform configuration file successfully updated to mn4-prace-raffa
Moving local files/dirs
Converting abs symlink to relative
Converting the absolute symlinks into relatives on platform marenostrum4
Converting absolute symlinks this can take a while depending on the experiment size
Moving remote files/dirs on marenostrum4
^[[32mFiles/dirs on marenostrum4 have been successfully offered^[[0m^[[39m
^[[32m[marenostrum4] from platforms configuration OK^[[0m^[[39m
Checking [transfer_node] from platforms configuration...
Project in platform configuration file successfully updated to cns61
Host in platform configuration file successfully updated to mn-dt-prace-raffa
Moving local files/dirs
^[[32mFiles/dirs on transfer_node have been successfully offered^[[0m^[[39m
^[[32m[transfer_node] from platforms configuration OK^[[0m^[[39m
Archiving in year 2020
Creating tar file ...
Tar file created!
^[[32mExperiment archived successfully^[[0m^[[39m
^[[32mThe experiment has been successfully offered.^[[0m^[[39m
Migrating experiment a28v
Moving local files/dirs
Experiment located in 2020 archive
Unpacking tar file ...
Unpacking finished
^[[32mExperiment a28v unarchived successfully^[[0m^[[39m
Local files/dirs have been successfully picked up
Checking configuration files...
platforms_a28v.conf OK
jobs_a28v.conf OK
autosubmit_a28v.conf OK
expdef_a28v.conf  OK
^[[32mConfiguration files OK
^[[0m^[[39m
Checking remote platforms
Copying remote files/dirs on marenostrum4
Copying from /gpfs/projects/bsc32/a28v/tmp_autosubmit_migrate/a28v to /gpfs/scratch/pr1eki00/pr1eki07/a28v
^[[33m[WARNING] Command cp -rP /gpfs/projects/bsc32/a28v/tmp_autosubmit_migrate/a28v /gpfs/scratch/pr1eki00/pr1eki07/a28v in mn4-prace-raffa warning: cp: cannot stat '/gpfs/projects/bsc32/a28v/tmp_autosubmit_migrate/a28v': No such file or directory
^[[0m^[[39m
^[[33m[WARNING] Command chmod 755 -R /gpfs/scratch/pr1eki00/pr1eki07/a28v in mn4-prace-raffa warning: chmod: cannot access '/gpfs/scratch/pr1eki00/pr1eki07/a28v': No such file or directory
^[[0m^[[39m
^[[32mFiles/dirs on marenostrum4 have been successfully picked up^[[0m^[[39m
^[[33m[WARNING] Command rm -R /gpfs/projects/bsc32/a28v/tmp_autosubmit_migrate/a28v in mn4-prace-raffa warning: rm: cannot remove '/gpfs/projects/bsc32/a28v/tmp_autosubmit_migrate/a28v': No such file or directory
^[[0m^[[39m
^[[32mThe experiment has been successfully picked up.^[[0m^[[39m
```

As you can see, I have an issue with remote files. As for the local ones, it seems to be ok.

@mcastril @dbeltrankyl

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Mar 3, 2020, 14:14

Hi @thomasarsouze , 

I think the issue is in the filesystem, there is a warning in the documentation:

> The temporary directory must be in the same filesystem.

Apparently the problem was during the offer process where the files couldn't be sent to `/gpfs/project` from `/gpfs/scratch` this is an issue from paramiko api itself.

Although, due changes in logs for others issues had as a side effect that the migrate revert mechanism  didn't notice this during the process

 The solution, per example, is to make the temp folder on your `/gpfs/scratch/bsc32/bsc32xxx/tmp_folder` instead of use `/gpfs/project`. Maybe I should change the readthedocs to make this more clear.

In any case, it seems that all the preparation were doing successfully so you can manually copy the remote files and it should work:

**remote platform**

* `chmod 777 -R /gpfs/scratch/cns61/cns61280/a28v`

* `mv /gpfs/scratch/cns61/cns61280/a28v <tmp_folder>`

* `cp -rP <tmp_folder> /gpfs/scratch/pr1eki00/pr1eki07/a28v`

* `chmod 755 -R /gpfs/scratch/pr1eki00/pr1eki07/a28v` (or the permissions that you want)

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Mar 3, 2020, 14:17

edited last msg, typos

**Comment by kinow** [comment]
In GitLab by @mcastril on Mar 3, 2020, 14:19

> Apparently the problem was during the offer process where the files couldn't be sent to `/gpfs/project` from `/gpfs/scratch` this is an issue from paramiko api itself. 

Sorry @thomasarsouze , I didn't remember there was this constraint.

> Maybe I should change the readthedocs to make this more clear. 

Maybe the problem is not the documentation itself, but we use to call GPFS a filesystem, and indeed in MN4 we have different filesystems.

```
gpfs_scratch                                                                  8.8P  7.9P  888T  91% /gpfs/scratch
gpfs_apps                                                                      20T   18T  2.7T  87% /gpfs/apps
gpfs_home                                                                      40T   39T  1.9T  96% /gpfs/home
gpfs_projects                                                                 4.4P  3.3P  1.1P  76% /gpfs/projects
```

**Comment by kinow** [comment]
In GitLab by @mcastril on Mar 3, 2020, 14:20

Maybe you can just modify it to state: "filesystem/partition" ?

I don't know if that can make a difference.

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Mar 3, 2020, 14:26

I will add use case examples maybe it will be more clear this way

**Comment by kinow** [comment]
In GitLab by @thomasarsouze on Mar 3, 2020, 15:31

>  **remote platform**
>
>* `chmod 777 -R /gpfs/scratch/cns61/cns61280/a28v`
>* `mv /gpfs/scratch/cns61/cns61280/a28v <tmp_folder>`
>* `cp -rP <tmp_folder> /gpfs/scratch/pr1eki00/pr1eki07/a28v`
>* `chmod 755 -R /gpfs/scratch/pr1eki00/pr1eki07/a28v` (or the permissions that you want)

Ok thanks, but will this be enough to preserve the links working in these directories ? This was already done during the preparation ?

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Mar 3, 2020, 15:35

/esarchive/autosubmit/ASlogs/20200303_110204_migrate_a28v.log
```
2020-03-03 11:02:04,399 Converting abs symlink to relative
2020-03-03 11:02:04,399 Converting the absolute symlinks into relatives on platform marenostrum4 
2020-03-03 11:02:13,115 Output var="$(realpath -s --relative-to="/gpfs/scratch/cns61/cns61280/a28v/19500101/fc0/inidata/ifs/macv2sp-data" "$(readlink "/gpfs/scratch/cns61/cns61280/a28v/19500101/fc0/inidata/ifs/macv2sp-data")")" && var=${var:3} && ln -sf $var "/gpfs/scratch/cns61/cns61280/a28v/19500101/fc0/inidata/ifs/macv2sp-data"  
var="$(realpath -s --relative-to="/gpfs/scratch/cns61/cns61280/a28v/19500101/fc0/inidata/ifs/cmip6-data" "$(readlink "/gpfs/scratch/cns61/cns61280/a28v/19500101/fc0/inidata/ifs/cmip6-data")")" && var=${var:3} && ln -sf $var "/gpfs/scratch/cns61/cns61280/a28v/19500101/fc0/inidata/ifs/cmip6-data"  
var="$(realpath -s --relative-to="/gpfs/scratch/cns61/cns61280/a28v/19500101/fc0/inidata/ifs/cmip5-data" "$(readlink "/gpfs/scratch/cns61/cns61280/a28v/19500101/fc0/inidata/ifs/cmip5-data")")" && var=${var:3} && ln -sf $var "/gpfs/scratch/cns61/cns61280/a28v/19500101/fc0/inidata/ifs/cmip5-data"  
var="$(realpath -s --relative-to="/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/ece2cmor/ece2cmor3/cmor_19500101_1_fc0-pa" "$(readlink "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/ece2cmor/ece2cmor3/cmor_19500101_1_fc0-pa")")" && var=${var:3} && ln -sf $var "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/ece2cmor/ece2cmor3/cmor_19500101_1_fc0-pa"  
var="$(realpath -s --relative-to="/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/util/ELPiN/bathymetries/nemo_bathy_meter_ORCA025.nc" "$(readlink "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/util/ELPiN/bathymetries/nemo_bathy_meter_ORCA025.nc")")" && var=${var:3} && ln -sf $var "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/util/ELPiN/bathymetries/nemo_bathy_meter_ORCA025.nc"  
var="$(realpath -s --relative-to="/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/util/ELPiN/bathymetries/nemo_bathy_meter_ORCA12.nc" "$(readlink "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/util/ELPiN/bathymetries/nemo_bathy_meter_ORCA12.nc")")" && var=${var:3} && ln -sf $var "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/util/ELPiN/bathymetries/nemo_bathy_meter_ORCA12.nc"  
var="$(realpath -s --relative-to="/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/util/ELPiN/bathymetries/nemo_bathy_meter_ORCA1.nc" "$(readlink "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/util/ELPiN/bathymetries/nemo_bathy_meter_ORCA1.nc")")" && var=${var:3} && ln -sf $var "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/util/ELPiN/bathymetries/nemo_bathy_meter_ORCA1.nc"  
var="$(realpath -s --relative-to="/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/util/ELPiN/bathymetries/nemo_bathy_meter_ORCA2.nc" "$(readlink "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/util/ELPiN/bathymetries/nemo_bathy_meter_ORCA2.nc")")" && var=${var:3} && ln -sf $var "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/util/ELPiN/bathymetries/nemo_bathy_meter_ORCA2.nc"  
var="$(realpath -s --relative-to="/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/nemo-3.6/CONFIG/ORCA12L75_LIM3_test/WORK/*.[Ffh]90" "$(readlink "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/nemo-3.6/CONFIG/ORCA12L75_LIM3_test/WORK/*.[Ffh]90")")" && var=${var:3} && ln -sf $var "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/nemo-3.6/CONFIG/ORCA12L75_LIM3_test/WORK/*.[Ffh]90"  
var="$(realpath -s --relative-to="/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/nemo-3.6/CONFIG/ORCA12L75_LIM3/WORK/*.[Ffh]90" "$(readlink "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/nemo-3.6/CONFIG/ORCA12L75_LIM3/WORK/*.[Ffh]90")")" && var=${var:3} && ln -sf $var "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/nemo-3.6/CONFIG/ORCA12L75_LIM3/WORK/*.[Ffh]90"  
var="$(realpath -s --relative-to="/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/xios-2/extern/netcdf4" "$(readlink "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/xios-2/extern/netcdf4")")" && var=${var:3} && ln -sf $var "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/xios-2/extern/netcdf4"  

2020-03-03 11:02:13,151 Output var="$(realpath -s --relative-to="/gpfs/scratch/cns61/cns61280/a28v/19500101/fc0/inidata/ifs/macv2sp-data" "$(readlink "/gpfs/scratch/cns61/cns61280/a28v/19500101/fc0/inidata/ifs/macv2sp-data")")" && var=${var:3} && ln -sf $var "/gpfs/scratch/cns61/cns61280/a28v/19500101/fc0/inidata/ifs/macv2sp-data"  
var="$(realpath -s --relative-to="/gpfs/scratch/cns61/cns61280/a28v/19500101/fc0/inidata/ifs/cmip6-data" "$(readlink "/gpfs/scratch/cns61/cns61280/a28v/19500101/fc0/inidata/ifs/cmip6-data")")" && var=${var:3} && ln -sf $var "/gpfs/scratch/cns61/cns61280/a28v/19500101/fc0/inidata/ifs/cmip6-data"  
var="$(realpath -s --relative-to="/gpfs/scratch/cns61/cns61280/a28v/19500101/fc0/inidata/ifs/cmip5-data" "$(readlink "/gpfs/scratch/cns61/cns61280/a28v/19500101/fc0/inidata/ifs/cmip5-data")")" && var=${var:3} && ln -sf $var "/gpfs/scratch/cns61/cns61280/a28v/19500101/fc0/inidata/ifs/cmip5-data"  
var="$(realpath -s --relative-to="/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/ece2cmor/ece2cmor3/cmor_19500101_1_fc0-pa" "$(readlink "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/ece2cmor/ece2cmor3/cmor_19500101_1_fc0-pa")")" && var=${var:3} && ln -sf $var "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/ece2cmor/ece2cmor3/cmor_19500101_1_fc0-pa"  
var="$(realpath -s --relative-to="/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/util/ELPiN/bathymetries/nemo_bathy_meter_ORCA025.nc" "$(readlink "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/util/ELPiN/bathymetries/nemo_bathy_meter_ORCA025.nc")")" && var=${var:3} && ln -sf $var "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/util/ELPiN/bathymetries/nemo_bathy_meter_ORCA025.nc"  
var="$(realpath -s --relative-to="/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/util/ELPiN/bathymetries/nemo_bathy_meter_ORCA12.nc" "$(readlink "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/util/ELPiN/bathymetries/nemo_bathy_meter_ORCA12.nc")")" && var=${var:3} && ln -sf $var "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/util/ELPiN/bathymetries/nemo_bathy_meter_ORCA12.nc"  
var="$(realpath -s --relative-to="/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/util/ELPiN/bathymetries/nemo_bathy_meter_ORCA1.nc" "$(readlink "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/util/ELPiN/bathymetries/nemo_bathy_meter_ORCA1.nc")")" && var=${var:3} && ln -sf $var "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/util/ELPiN/bathymetries/nemo_bathy_meter_ORCA1.nc"  
var="$(realpath -s --relative-to="/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/util/ELPiN/bathymetries/nemo_bathy_meter_ORCA2.nc" "$(readlink "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/util/ELPiN/bathymetries/nemo_bathy_meter_ORCA2.nc")")" && var=${var:3} && ln -sf $var "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/util/ELPiN/bathymetries/nemo_bathy_meter_ORCA2.nc"  
var="$(realpath -s --relative-to="/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/nemo-3.6/CONFIG/ORCA12L75_LIM3_test/WORK/*.[Ffh]90" "$(readlink "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/nemo-3.6/CONFIG/ORCA12L75_LIM3_test/WORK/*.[Ffh]90")")" && var=${var:3} && ln -sf $var "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/nemo-3.6/CONFIG/ORCA12L75_LIM3_test/WORK/*.[Ffh]90"  
var="$(realpath -s --relative-to="/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/nemo-3.6/CONFIG/ORCA12L75_LIM3/WORK/*.[Ffh]90" "$(readlink "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/nemo-3.6/CONFIG/ORCA12L75_LIM3/WORK/*.[Ffh]90")")" && var=${var:3} && ln -sf $var "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/nemo-3.6/CONFIG/ORCA12L75_LIM3/WORK/*.[Ffh]90"  
var="$(realpath -s --relative-to="/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/xios-2/extern/netcdf4" "$(readlink "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/xios-2/extern/netcdf4")")" && var=${var:3} && ln -sf $var "/gpfs/scratch/cns61/cns61280/a28v/auto-ecearth3/sources/sources/xios-2/extern/netcdf4"  
```



According to this log seems that step worked

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Mar 3, 2020, 15:37

Note: Since there is not a exp folder during the process the log is stored in this `/esarchive/autosubmit/ASlogs/`, same happens to other commands like expid

**Comment by kinow** [comment]
In GitLab by @thomasarsouze on Mar 3, 2020, 16:55

When running this:

>  * `mv /gpfs/scratch/cns61/cns61280/a28v <tmp_folder>`

the command crashes. This already happened to me. I guess this comes from the very big runtime dir that I have (probably > 1To). 

How am I supposed to resume the transfer : rsync -ar ? Is it ok for the links ?

**Comment by kinow** [comment]
In GitLab by @mcastril on Mar 3, 2020, 17:56

I recommend to do big data movements in dt (I think there's no 10 minute limit there) and even better through dt commands (dtmv, dtcp, dtrsync) that are up to 10 times faster.

**Comment by kinow** [comment]
In GitLab by @mcastril on Mar 3, 2020, 17:56

The key is how to be explicative enough for BSC-ES users, while preserving agnosticism for the rest of the world.

**Comment by kinow** [comment]
In GitLab by @etiennesky on Mar 3, 2020, 18:14

dtmv is indeed the preffered method, but standard mv should work if running on the dt node, instead of login nodes.

**Comment by kinow** [comment]
In GitLab by @thomasarsouze on Mar 5, 2020, 10:21

Hi,

one comment from the migrate command. 

I think that the re-linking to relative path is nice for the inidata folder. However, this is not satisfactory for some other files. I'm thinking of the files in the CONFIGURATION/XXX/WORK directory in NEMO that now points to files from a previous PRACE project that no longer exists. These ones wouldn't have to be relinked. Maybe there is the same issue for other model files, will need to check...

Overall this is not very important as the executables don't change.

**Comment by kinow** [comment]
In GitLab by @mcastril on Mar 6, 2020, 12:33

Thanks for pointing this out @thomasarsouze , it has not been reported previously.

I see that the NEMO binaries are also not being linked. @dbeltrankyl , do you know what could it happen? It is a matter of tree depth or maybe the process was stopped?

**Comment by kinow** [comment]
In GitLab by @mcastril on May 21, 2020, 13:42

Hi @dbeltrankyl , we have skipped this issue in our meetings, as it is not in the Autosubmit project.

I am moving it.

**Comment by kinow** [comment]
In GitLab by @mcastril on May 21, 2020, 13:42

moved from auto-ecearth3#1217

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on May 21, 2020, 14:56

Hi ,

I totally forgot this issue, when they aren't inside autosubmit git I tend to forget about it since I use it for the refresh which ones are in todo list, apologies-

>I see that the NEMO binaries are also not being linked. @dbeltrankyl , do you know what could it happen? It is a matter of tree depth or maybe the process was stopped?

I guess the issue was that the process was stopped the maximum time is 1h maybe we should increase it? , the tree depth shouldn't be a problem at least in the test that I did time ago, but maybe it wasn't covering all cases since there a huge amount of possible scenarios.

Maybe we should perform tests with  larger experiments and different configurations and see what happens

**Comment by kinow** [comment]
In GitLab by @mcastril on Sep 29, 2020, 11:12

mentioned in issue auto-ecearth3#1412

**Comment by kinow** [comment]
In GitLab by @mcastril on Nov 3, 2020, 16:20

I verified the status of the links for the current test and this is the only two broken links that I found:

```
./auto-ecearth3/sources/sources/xios-2.5/extern/netcdf4
./auto-ecearth3/sources/sources/nemo-3.6/CONFIG/ORCA1L75_LIM3_PISCES/WORK/*.[Ffh]90
```

The second is usually already broken and I think it comes from a wrong link creation. The first is only used in XIOS compilation, so I don't think it is needed either.

**Comment by kinow** [comment]
In GitLab by @etiennesky on Nov 3, 2020, 16:28

this run includes lpjg right?

**Comment by kinow** [comment]
In GitLab by @mcastril on Nov 3, 2020, 16:53

Yes, it comes from https://earth.bsc.es/gitlab/es/auto-ecearth3/-/issues/879

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Jan 12, 2021, 12:51

There is still a test pending on auto-ecearth3#879, I'll do it this week

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on Mar 3, 2021, 13:14

After some tweaks on migrate, 

We are hoping that we're doing the last test also using RES account from julian to @dbeltrankyl RES user.

at auto-ecearth3#879

**Comment by kinow** [comment]
In GitLab by @etiennesky on Mar 3, 2021, 13:26

careful that the cns04 accounts  only have access to class_c queue for the next few days, and will be completely disabled after that.

**Comment by kinow** [comment]
In GitLab by @mcastril on Mar 3, 2021, 15:47

@dbeltrankyl was added to the last one granted to Oriol, so it should last at least some months.

**Comment by kinow** [comment]
In GitLab by @dbeltrankyl on May 17, 2021, 13:33

Migrate feature was tested, on some other issue but not sure which one was, @mcastril. 

Anyway we can close this one

**Comment by kinow** [comment]
In GitLab by @mcastril on May 17, 2021, 13:37

It's in the Auto-EC-Earth board. I have pending to start writing the wiki in the Auto-EC-Earth board too, but you can close.
